src/modules/openrccfg/openrccfg.conf

```diff
@@ -0,0 +1,8 @@
+---
+services:
```

Comment:
Most modules have documentation on the keys and possible values. Here, something like 

> *services* is a map with one of more object keys (e.g. *add*) that correspond to
> the first argument of rc-update. The following keys are in use: *add*, *del*.
> The value of each of those keys is a list of service maps; each of those service
> maps has a key *name* and a key *runlevel*. The named service is added, or
> deleted, from the given runlevel. To add or remove a service from multiple
> runlevels, list it multiple times with different runlevels.

Reply:
altough OpenRC allows adding same service into multiple runlevels, this is a bad behaviour...OpenRC is a dependency based init system, everything must be started at a specific point...I don't want to encourage people to add say NetworkManager to sysinit runlevel, which is only intended to start critical services (udev, devfs, sysfs)...

that's why I really think this module should only limit itself to the default runlevel...other runlevels (sysinit, boot) are critical to bring the box up, so the installer must not touch them...